mds/apis/prv_api/vehicles/__init__.py

```diff
@@ -0,0 +1 @@
+from .views import DeviceViewSet  # noqa
```

Comment:
What's the purpose of this line ?

Reply:
Preserve import compatibility, so we can `from mds.apis.prv_apis.vehicles import DeviceViewSet` just as before.